doc/source/getting_started/installation.rst

```diff
@@ -1,92 +1,69 @@
-.. _installation:
+.. _ref_installation:
 
 ============
 Installation
 ============
 
-The ``ansys-fluent-core`` package supports Python 3.9 through
-Python 3.12 on Windows and Linux.
 
-.. note::
-   This page provides information for quickly installing and launching
-   the ``ansys-fluent-core`` package. Additional PyFluent packages, which
-   are described later on this page, can also be installed. For
-   information on installing all PyFluent packages in a virtual environment,
-   see :ref:`faqs_install` in :ref:`faqs`.
+PyFluent Installation
+---------------------
 
-Install the package
--------------------
-Install the latest ``ansys-fluent-core`` package from
-`PyPi <https://pypi.org/project/ansys-fluent-core/>`_ with this code:
+PyFluent supports Python 3.9 through Python 3.12 on Windows, Mac OS and Linux.
+
+PyFluent can be installed, along with all its optional dependencies, using:
 
 .. code::
 
    pip install ansys-fluent-core
 
-If you plan on doing local *development* of PyFluent with Git, install the
-latest ``ansys-fluent-core`` package with this code:
 
+Development Installation
+------------------------
+
+The PyFluent source repository is available on GitHub. You can clone the repository using:
 .. code:: console
 
    git clone https://github.com/ansys/pyfluent.git
-   cd pyfluent
-   pip install pip -U
-   pip install -e .
-   python codegen/allapigen.py  # Generates the API files
-
-
-Any changes that you make locally are reflected in your setup after you restart
-the Python kernel.
-
-Launch Fluent
--------------
-To launch Fluent from PyFluent, use the :func:`launch_fluent() <ansys.fluent.core.launcher.launcher.launch_fluent>`
-method:
-
-.. code:: python
 
-  import ansys.fluent.core as pyfluent
-  solver = pyfluent.launch_fluent(precision="double", processor_count=2, mode="solver")
-  solver.health_check.is_serving
+Once you have cloned the repository, run the following commands. Installing with the
```

Comment:
The sentence "run the following commands" seems a bit misplaced here, as the commands don't follow that sentence, if that makes sense.
```suggestion
Once you have cloned the repository, you can install PyFluent. Installing with the
```